vllm/engine/multiprocessing/engine.py

```diff
@@ -110,25 +111,39 @@ def dead_error(self) -> BaseException:
             return ENGINE_DEAD_ERROR()
 
     @classmethod
-    def from_engine_args(cls, engine_args: AsyncEngineArgs,
-                         usage_context: UsageContext, ipc_path: str):
-        """Creates an MQLLMEngine from the engine arguments."""
+    def from_vllm_config(cls, vllm_config: VllmConfig,
```

Comment:
iirc, `from_engine_args` is a user-facing api, and lots of people use it. if possible, we should try to keep it?

Reply:
it still exists below